app/src/features/settings/components/BillingTeam/types.ts

```diff
@@ -30,3 +30,8 @@ export interface BillingTeamMember {
   role: BillingTeamRoles;
   joiningDate: number;
 }
+
+export enum BillingTeamJoinRequestAction {
+  ACCEPT = "accept",
+  REJECT = "reject",
+}
```

Comment:
No need to create enums for 1 time use string literals. Can create local method constants only